src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -40,7 +40,9 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
   }
 
   public componentDidUpdate = (prevProps: IPermissionProps) => {
-    if ((prevProps.sample !== this.props.sample) || (prevProps.permissionsPanelOpen !== this.props.permissionsPanelOpen) || (prevProps.permissionModeType !== this.props.permissionModeType)) {
+    if (prevProps.sample !== this.props.sample
+      || prevProps.permissionsPanelOpen !== this.props.permissionsPanelOpen
+      || prevProps.permissionModeType !== this.props.permissionModeType) {
```

Comment:
```suggestion
        || prevProps.permissionsPanelOpen !== this.props.permissionsPanelOpen
        || prevProps.permissionModeType !== this.props.permissionModeType) {
```
The extra indentation is an intentional formatting for if statements / functions of the like